draft-ietf-tls-esni.md

```diff
@@ -994,6 +974,72 @@ The "ech_is_inner" is expected to have an empty payload. If the payload is
 non-empty (i.e., the length of the "extension_data" field is non-zero) then the
 backend server MUST abort the handshake with an "illegal_parameter" alert.
 
+## Sending HelloRetryRequest {#server-hrr}
+
+If the backend server cannot complete the handshake with the parameters provided
+in the ClientHelloInner, then it constructs a HelloRetryRequestInner message in
+the usual way and forwards it to the client-facing server. The client-facing
```

Comment:
The usual way an HRR is constructed?

Reply:
Eh, perhaps s/usual way/as described in {{RFC8446}}/.

Reply:
Even that is confusing because there is no usual way to construct a "HelloRetryRequestInner." I would say something like "in the usual way a HelloRetryRequest is constructed"